kyuubi-server/src/test/scala/org/apache/kyuubi/operation/thrift/http/KyuubiOperationThriftHttpKerberosAndPlainAuthSuite.scala

```diff
@@ -49,19 +58,46 @@ class KyuubiOperationThriftHttpKerberosAndPlainAuthSuite
     UserGroupInformation.setConfiguration(config)
     assert(UserGroupInformation.isSecurityEnabled)
 
-    KyuubiConf().set(KyuubiConf.AUTHENTICATION_METHOD, Seq("KERBEROS", "LDAP", "CUSTOM"))
+    KyuubiConf().set(KyuubiConf.AUTHENTICATION_METHOD, Seq("KERBEROS", "CUSTOM", "LDAP"))
       .set(KyuubiConf.SERVER_KEYTAB, testKeytab)
       .set(KyuubiConf.SERVER_PRINCIPAL, testPrincipal)
       .set(KyuubiConf.AUTHENTICATION_LDAP_URL, ldapUrl)
       .set(KyuubiConf.AUTHENTICATION_LDAP_BASE_DN, ldapBaseDn.head)
       .set(
         KyuubiConf.AUTHENTICATION_CUSTOM_CLASS,
         classOf[UserDefineAuthenticationProviderImpl].getCanonicalName)
+      .set(
+        KyuubiConf.AUTHENTICATION_CUSTOM_BEARER_CLASS,
+        classOf[UserDefineAuthenticationProviderImpl].getCanonicalName)
       .set(KyuubiConf.SERVER_SPNEGO_KEYTAB, testKeytab)
       .set(KyuubiConf.SERVER_SPNEGO_PRINCIPAL, testSpnegoPrincipal)
   }
 
   override protected def getJdbcUrl: String =
     s"jdbc:hive2://${server.frontendServices.head.connectionUrl}/default;transportMode=http;" +
       s"httpPath=cliservice"
+
+  test("test with valid CUSTOM http bearer authentication") {
+    val conn = DriverManager.getConnection(JdbcUrlWithValidBearerToken)
```

Comment:
FYI:    
```
withSessionConf(Map("bearerToken" -> "token"))()() {
     val conn = DriverManager.getConnection(jdbcUrlWithConf)
  }
```

Reply:
I tried pass it via session conf as suggested, but seems in KyuubiConnection.scala
`sessConfMap = connParams.getSessionVars();`
it couldn't get bearToken variable.

Reply:
Did you use `jdbcUrlWithConf`? 

It will append the sessionConf to url.

Reply:
Yeah I figured out, i was passing httpPath=cliservicebearerToken=token without the ";" so got null.